tests/Acceptance/DependencyInjection/ConfigurationTest.php

```diff
@@ -34,9 +35,10 @@ public function validTest(array $configs, string $expectedConfigName): void
 
     /**
      * @test
+     *
      * @dataProvider provideInvalidConfig
      */
-    public function invalid(array $config, ?string $expectedMessage = null): void
+    public function invalid(array $config, string $expectedMessage = null): void
```

Comment:
we are used to leaving `?` for nullable values

Reply:
This was not my change. Tell this to your CS fixer :-)

Reply:
🙈 please, revert this, skip checking by adding `-n` on commiting. I will fix CS fixer then. Thanks

Reply:
https://github.com/answear/luigis-box-bundle/pull/47 - merged fix for this

Reply:
done